nms/app/packages/magmalte/app/components/context/RefreshContext.js

```diff
@@ -164,6 +193,17 @@ async function fetchRefreshState(props: FetchProps) {
     });
 
     return gateways;
+  } else if (type === RefreshTypeEnum.FEG_GATEWAY) {
+    const fegGateways = await FetchFegGateways({
+      id: id,
+      networkId,
+      enqueueSnackbar,
+    });
+    const [health, activeFegGatewayId] = await Promise.all([
+      getFegGatewaysHealthStatus(networkId, fegGateways || {}, enqueueSnackbar),
```

Comment:
`fegGateways || {}` shouldn't be necessary here. Getting the health is arbitrary if there aren't any `fegGateways`.